cpp/src/Slice/Scanner.l

```diff
@@ -552,154 +548,150 @@ floating_literal    (({fractional_constant}{exponent_part}?)|((\+|-)?{dec}+{expo
 
 namespace Slice
 {
-
-// Check if an identifier looks like a keyword.
-// If the identifier is a keyword, return the
-// corresponding keyword token; otherwise, return
-// an identifier token.
-int checkKeyword(string& id)
-{
-    const StringTokenMap::const_iterator pos = keywordMap.find(id);
-    if(pos != keywordMap.end())
+    // Check if an identifier looks like a keyword.
+    // If the identifier is a keyword, return the
+    // corresponding keyword token; otherwise, return
+    // an identifier token.
+    int checkKeyword(string& id)
     {
-        if(pos->first != id)
+        const StringTokenMap::const_iterator pos = keywordMap.find(id);
+        if (pos != keywordMap.end())
         {
-            currentUnit->error("illegal identifier: `" + id + "' differs from keyword `" + pos->first +
-                        "' only in capitalization");
-            id = pos->first;
+            if (pos->first != id)
+            {
+                currentUnit->error("illegal identifier: `" + id + "' differs from keyword `" + pos->first +
```

Comment:
minor: this is aligned differently in the generated version.

Reply:
Good catch and fair enough!
Fixed!